empress/support_files/js/emperor-callbacks.js

```diff
@@ -64,15 +64,12 @@ plotView.on("click", function (name, object) {
 });
 
 plotView.on("select", function (samples, view) {
+    // remove any strangling observers
```

Comment:
```suggestion
    // remove any ongoing observers
```
... I don't think "strangling" is the correct word for this, but please correct me if i'm wrong!

Reply:
Naming things is the bane of my existence lol. I actually  meant `dangling observer` :laughing: My logic was as follows:
Every time samples are selected or values are clicked on in emperor the callback functions are called. Thus, if a value is clicked on before the timeout event is called, the new observer will supersede the old observer and the old observer can be thought of as being de-referenced  hence a dangling observer.

However, I like `ongoing` much better since the old observer is not actually de-referenced and will still be called unless explicitly unregistered.